content/en/docs/Reference/build-from-scratch.md

```diff
@@ -179,4 +179,30 @@ docker run -v --net host \
 {{% /tab %}}
 {{< /tabpane >}}
 
-This example is available in the `examples/byoi/fedora` directory of the [Kairos repository](https://github.com/kairos-io/kairos/tree/master/examples/byoi/fedora), where you can run `build.sh` to reproduce it.
\ No newline at end of file
+This example is available in the `examples/byoi/fedora` directory of the [Kairos repository](https://github.com/kairos-io/kairos/tree/master/examples/byoi/fedora), where you can run `build.sh` to reproduce it.
+
+## FIPSs compliant flavors
```

Comment:
```suggestion
## FIPS compliant flavors
```

I don't think the s is needed for the plural

Reply:
true :)

Reply:
https://github.com/kairos-io/kairos-docs/pull/40/commits/b5aecff9d4a93752396bbf65970107d605171277